inference/core/workflows/core_steps/common/utils.py

```diff
@@ -421,3 +423,81 @@ def run_in_parallel(tasks: List[Callable[[], T]], max_workers: int = 1) -> List[
 
 def _run(fun: Callable[[], T]) -> T:
     return fun()
+
+
+def convert_gaze_detections_to_sv_detections_and_angles(
```

Comment:
this seems like an odd place for this to live.  Why not closer / directly in the file with the rest of the block?

Reply:
Hm, no particular reason.. Just saw that all `convert_...` methods lived inside this file, thought I'd be following the repo pattern